x-pack/plugins/serverless_observability/public/plugin.ts

```diff
@@ -25,8 +26,11 @@ export class ServerlessObservabilityPlugin
 
   public start(
     _core: CoreStart,
```

Comment:
The convention is to use an `_` prefix for unused variables, that might be needed for reference or in the future. Using the `_` silences a compile error when the variable isn't used. But when it is used, it should just have the standard name of `core`.

Reply:
Addressed here: https://github.com/elastic/kibana/pull/157360/commits/aebe5f9084f6aed1890ef817e6e9784028f1a9fc